content/blog/pulumi-2-0-update/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "An Update on Pulumi 2.0"
+authors: ["joe-duffy"]
+tags: ["Pulumi-News"]
+date: "2020-03-06"
+meta_desc: "Learn about some of Pulumi's progress rounding out our .NET and Go SDKs in addition to infrastructure validation using testing and Policy as Code."
+meta_image: "pulumi-1-0.png"
```

Comment:
Nit: Are you planning to use a different meta image? Not sure why this is using a file named "1-0" for an update on 2.0.

Reply:
I'm just reusing the same header preview image from the 1.0 launch since we don't have the new 2.0 copy.